package.json

```diff
@@ -30,13 +30,9 @@
     "prettier-plugin-ember-template-tag": "^1.1.0",
     "release-plan": "^0.9.0"
   },
+  "packageManager": "pnpm@9.1.1",
   "volta": {
     "node": "18.20.0",
     "pnpm": "9.1.1"
-  },
-  "pnpm": {
-    "overrides": {
-      "@types/eslint": "^7.0.0"
```

Comment:
There is no usage of this package, hence removing ✂️. This may be related to `The current "overrides" configuration doesn't match the value found in the lockfile` error from the [failed action](https://github.com/zonkyio/ember-liveagent/actions/runs/9959203046/job/27515485643) 🤞